umm-lib/resources/schema/dif10/UmmCommon_1.2.xsd

```diff
@@ -457,6 +457,9 @@ technical contact
             <xs:enumeration value="PLANNED"/>
             <xs:enumeration value="IN WORK"/>
             <xs:enumeration value="COMPLETE"/>
+            <xs:enumeration value="PREPRINT"/>
+            <xs:enumeration value="INREVIEW"/>
+            <xs:enumeration value="SUPERSEDED"/>
```

Comment:
I think this is OK, we need to be backwards compatible so that users of dif 10 don't break.

Reply:
Also, I don't think we need to change umm-lib anymore. All validations except for granules occur in umm-spec-lib.  I have been wanted to delete these, but never get the time.

Reply:
keep it.